docker-plugin/src/test/java/com/nirima/jenkins/plugins/docker/ws/MappedFsWorkspaceBrowserTest.java

```diff
@@ -0,0 +1,84 @@
+package com.nirima.jenkins.plugins.docker.ws;
+
+import com.nirima.jenkins.plugins.docker.action.DockerBuildAction;
+import hudson.FilePath;
+import hudson.model.Job;
+import hudson.model.Run;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.apache.commons.lang.RandomStringUtils.randomAlphabetic;
+import static org.hamcrest.Matchers.both;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.endsWith;
+import static org.hamcrest.Matchers.nullValue;
+import static org.hamcrest.Matchers.startsWith;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.when;
+
+/**
+ * User: lanwen
+ * Date: 22.05.15
+ * Time: 17:01
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class MappedFsWorkspaceBrowserTest {
+
+    public static final String FS_MAPPING = "/mapping";
+    public static final String JOB_NAME = "job-name";
+
+    @Mock
+    private Job job;
+
+    @Mock
+    private Run run;
+
+    @Test
+    public void shouldReturnWsMappingOnFieldInAction() throws Exception {
+        MappedFsWorkspaceBrowser mappedFsWorkspaceBrowser = new MappedFsWorkspaceBrowser();
+
+        when(job.getLastBuild()).thenReturn(run);
+        when(job.getName()).thenReturn(JOB_NAME);
+        when(run.getAction(DockerBuildAction.class))
```

Comment:
don't think it should be done - because of it needs to check how it gets the action from jenkins (it needs mocks) and because of content of such static method will be equal the content of inner `if` (its overhead). Mocks here is simple and easy to use